dffml/distribuited/orchestrator.py

```diff
@@ -0,0 +1,427 @@
+import abc
+import json
+import asyncio
+import secrets
+import traceback
+from queue import Queue
+from dataclasses import field
+from typing import Dict, List, Any
+from contextlib import asynccontextmanager, AsyncExitStack, ExitStack
+
+from nats.aio.client import Client as NATS
+
+from ..base import config, BaseConfig
+from ..util.asynchelper import aenter_stack
+from ..df.types import DataFlow, Operation, Definition, Input
+from ..df.memory import (
+    MemoryRedundancyChecker,
+    MemoryLockNetwork,
+    MemoryOperationImplementationNetwork,
+    MemoryOperationImplementationNetworkContext,
+    MemoryOperationNetworkContext,
+    MemoryOperationNetwork,
+)
+from ..df.base import (
+    BaseOrchestratorContext,
+    BaseOperationNetworkContext,
+    BaseOperationNetwork,
+    BaseDataFlowObject,
+    BaseDataFlowObjectContext,
+    BaseOperationImplementationNetworkContext,
+)
+
+
+# Subjects
+RegisterWorkerNode = "RegisterWorkerNode"
+ConnectToOrchestratorNode = "ConnectToOrchestratorNode"
+OrchestratorNodeReady = "OrchestratorNodeReady"
+
+####
+
+NBYTES_UID = 6
```

Comment:
Let's take this out of the http service and put it in util somewhere, maybe `dffml/util/default.py` then use it in this file

https://github.com/intel/dffml/blob/ad2286448b18ccd2b844e01ce02b8c1712d02e8d/service/http/dffml_service_http/routes.py#L41-L44